testing/test_dilateGT.py

```diff
@@ -0,0 +1,99 @@
+import os
+import numpy as np
+import time
+import sys
+import matplotlib.pyplot as plt
+
+import torch.backends.cudnn as cudnn
+from torch.utils.data import DataLoader
+from torchvision import transforms
+
+from medicaltorch.filters import SliceFilter
+from medicaltorch import datasets as mt_datasets
+from medicaltorch import transforms as mt_transforms
+from torch import optim
+
+from tqdm import tqdm
+
+from ivadomed import loader as loader
+from ivadomed import models
+from ivadomed import losses
+from ivadomed.utils import *
+import ivadomed.transforms as ivadomed_transforms
+
+cudnn.benchmark = True
+
+GPU_NUMBER = 5
+BATCH_SIZE = 8
+DROPOUT = 0.4
+BN = 0.1
+N_EPOCHS = 10
+INIT_LR = 0.01
+PATH_BIDS = 'testing_data/'
```

Comment:
throughout the code i would remove "/" to anticipate Windows compatibility and use `os.path.join()` that can deal with OS-specific separators